apollo-router/src/spec/query.rs

```diff
@@ -466,9 +466,9 @@ impl Query {
             },
             executable::Type::Named(name) if name == "Int" => {
                 let opt = if input.is_i64() {
-                    input.as_i64().and_then(|i| i32::try_from(i).ok())
+                    input.as_i64()
                 } else if input.is_u64() {
-                    input.as_i64().and_then(|i| i32::try_from(i).ok())
+                    input.as_i64()
```

Comment:
Should we add logging so we know what columns expose this? Tanner added it to sportsbook but if another team federates before we can revert this the router would be a common spot